lib/rls/api.rb

```diff
@@ -33,6 +33,36 @@ def player(id, platform = Platform::Steam)
       Player.new(response)
     end
 
+    # Retrieve a batch of players
```

Comment:
We could be a little more detailed here ; perhaps, "Retrieves a batch of up to 10 players at a time from different platforms"